docs/package_list.md

```diff
@@ -1,44 +1,44 @@
 | Package name                                                  | Package hash                                                  |
 | ------------------------------------------------------------- | ------------------------------------------------------------- |
 | protocol/valory/abci/0.1.0                                    | `bafybeiaw3tzlg3rkvnn5fcufblktmfwngmxugn4yo7pyjp76zz6aqtqcay` |
-| connection/valory/abci/0.1.0                                  | `bafybeidqqsef6pzg5qn4z7nmw2cwukudnqhtgrtsw3skbi3e2mvc4bgmv4` |
+| connection/valory/abci/0.1.0                                  | `bafybeiav3mywaeik35huowg6vjekllqiaeorxlc6eesc6aftlb7msh6mdu` |
```

Comment:
@dvilelaf Looks like there is some issue with the `check_doc_ipfs_hashes.py`, it cannot detect the bad hashes in the `packages_list.md`, neither it fixes it when using the `--fix` flag, please look into this once you're back

Reply:
Issue for the same https://github.com/valory-xyz/open-autonomy/issues/1515